### DISCUSSION TITLE
WXS Fragment to create subfolders/files in install directory

### DISCUSSION THREAD

**Discussion by LickABrick** [tauri-apps/tauri#11196] [Q&A]
Hi I can't get my Wix fragment working to create an aditional folder in the install directory. The fragment looks like this:

<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
  <Fragment>
    <DirectoryRef Id="APPLICATIONFOLDER">
      <Directory Id="MySubfolder" Name="MySubfolder" />
    </DirectoryRef>

    <Component Id="MySubfolderComponent" Guid="*">
      <CreateFolder Directory="MySubfolder" />
      <RemoveFolder Id="RemoveMySubfolder" Directory="MySubfolder" On="uninstall"/>
    </Component>
  </Fragment>
</Wix>

My config contains:

`"windows": {
        "certificateThumbprint": null,
        "digestAlgorithm": "sha256",
        "timestampUrl": "",
        "wix": {
          "fragmentPaths": ["folder-fragment.wxs"],
          "componentRefs": [
            "MySubfolderComponent"
          ]
        }
      }`

I get an error `Error failed to bundle project: error running candle.exe`

**Comment by FabianLars** [comment]
> I get an error Error failed to bundle project: error running candle.exe

If you run tauri build with the --verbose flag you will get the full wix logs which should hopefully help to debug this.
